### PR TITLE
feat: added title to s51 advice api

### PR DESCRIPTION
**Pull request by Yakitrak**
## Describe your changes

Ticket: https://pins-ds.atlassian.net/browse/ASB-2196

Adds title field to S51 advice both endpoints

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/applications-service-api/src/utils/advice.mapper.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ const mapCommonAdviceFieldsToApi = (advice) => {
 		dateEnquiryReceived: advice?.enquiryDate,
 		dateAdviceGiven: advice?.adviceDate,
 		dateLastModified: advice?.modifiedAt,
-		dateCreated: advice?.createdAt
+		dateCreated: advice?.createdAt,
+		title: advice?.title
```
Comment (Philip-Philippou1) [inline packages/applications-service-api/src/utils/advice.mapper.js:37]: We should probably update swagger spec:
.../packages/applications-service-api/api/openapi.yaml

Comment (Yakitrak) [reply packages/applications-service-api/src/utils/advice.mapper.js:37]: https://github.com/Planning-Inspectorate/applications-service/pull/999/commits/4aba185874c6bd78c72ccc127f1dc0e5d8256761

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**